dynawo/sources/ModelicaCompiler/DYNCompileModelicaModel.cpp

```diff
@@ -457,8 +457,8 @@ bool verifySharedObject(const string& library) {
     // constructor
   }
   catch(const std::exception& e) {
-    std::cerr << e.what() << '\n';
-    printf(" DYNCompileModelicaModel: could not open .so by boost dll.");
+    std::cerr << e.what() << std::endl;
+    std::cout << " DYNCompileModelicaModel: could not open .so by boost dll." << std::endl;
```

Comment:
Maybe we could the library name as it is available?

Reply:
OK